nss-c31-unusual-prestige/nutshell
# Homepage

# Description
Created homepage and registration form that saves new users.

Applies to issue # 6

## Type of change

Please delete options that are not relevant.



# Testing Instructions

Checkout branch and run grunt 
put this into main.js

import apiManager from "./apiManager.js";
import newsFeedDOM from "./newsFeed/newsDomManage.js"
import newsForm from "./newsFeed/newsForm.js"
import newsEventHandler from "./newsFeed/newsEventHandler.js"
import homepage from "./Homepage/homePage.js"

homepage.homePage()


make sure when register button is clicked form appears and data is saved when register save button is clicked.


# Checklist:
